CHANGELOG.md

```diff
@@ -6,9 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 
 ## Unreleased
 
+### Added
+- `map` Added possibility to set the zoom level of the `aui-leaflet-locate-control`.
+
+### Changed
+- [BREAKING] removed rxjs-compat dependency
```

Comment:
for a breaking change we should respect semver and release a new major, with a maintenance branch for the old major